### ISSUE TITLE
FDS Validation Guide Table 3.10 and 3.11 missing NIST Smoke Alarm Tests entries

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#2146]
```
Nightly build document retrieved 6/16/2014.  Talbe 3.10, Summary of important experimental
parameters, p 60 and Table 3.11, Summary of important numerical parameters, p 61 are
both missing an entry for the NIST Smoke Alarm tests included in the V&V data.

```

Original issue reported on code.google.com by `rpeacoc` on 2014-06-16 19:26:53

**Comment by gforney** [comment]
```
Thanks. I'll add it.
```

Original issue reported on code.google.com by `mcgratta` on 2014-06-16 19:40:30

**Comment by gforney** [comment]
```
Should be in the next nightly build.
```

Original issue reported on code.google.com by `mcgratta` on 2014-06-17 21:32:42

**Comment by gforney** [comment]
```
Checked the Validation Guide released with 6.1.1.
```

Original issue reported on code.google.com by `mcgratta` on 2014-07-18 20:57:50
